app/furniture/marketplace/product.rb

```diff
@@ -14,6 +13,9 @@ class Product < ApplicationRecord
     has_many :cart_products, inverse_of: :product, dependent: :destroy
     has_many :carts, through: :cart_products, inverse_of: :products
 
+    has_many :ordered_products, inverse_of: :product, dependent: :destroy
```

Comment:
This makes me think that maybe we don't want to be able to destroy Products that have been Ordered!

Reply:
so we'll always have to keep a record of any products in the database if anyone has ordered them even if the product is removed from the Marketplace/is no longer for sale?

Guess that's okay. Shouldn't take up too much database space right?

Reply:
Yea, it would be scary if a `Product` getting destroyed the `Order`s `OrderedProducts` because then how would `Vendor`s or `Shopper`s be able to see what they bought when?

Reply:
(Note: our hope is that Database costs are covered on a per-space basis; but I'm not entirely sure how we would go about doing that yet :D)